app/javascript/components/molecules/StatusDropdown.js

```diff
@@ -1,37 +1,85 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import { components } from 'react-select'
+import classNames from 'classnames'
 
-import { LEASE_UP_STATUS_OPTIONS, LEASE_UP_SUBSTATUS_OPTIONS, getLeaseUpStatusClass } from '~/utils/statusUtils'
+import { LEASE_UP_STATUS_OPTIONS, LEASE_UP_STATUS_VALUES } from '~/utils/statusUtils'
 import Dropdown from '../molecules/Dropdown'
 
-const StatusDropdown = ({ status, onChange, buttonClasses, menuClasses, wrapperClasses, styles, disabled, subStatus, showSubStatus, prompt }) => {
-  let dropdownButtonClasses = !showSubStatus ? [getLeaseUpStatusClass(status)] : []
-  if (buttonClasses) dropdownButtonClasses.push(...buttonClasses)
+const renderStatusOption = ({ value, label, style }) => {
+  return (
+    <li className={classNames('dropdown-menu_item', style)}>
+      <a>{label}</a>
+    </li>
+  )
+}
+
+const StatusDropdown = ({
+  status,
+  onChange,
+  disabled,
+  placeholder,
+  size,
+  expand
+}) => {
+  const buttonClasses = [
+    'button',
+    'dropdown-button',
+    'has-icon--right',
+    'text-align-left',
+    { 'expand': expand },
+    { 'tiny': size === 'tiny' },
+    { 'small': size === 'small' }
+  ]
+  const renderStatusToggle = ({ children, getValue, ...props }) => {
+    const val = getValue()[0]
+
+    return (
+      <button
```

Comment:
if all of our styled dropdowns are going to be buttons then it would make sense to make more generalized. The differences between substatus and status dropdowns are just a couple of classes